shared/metadata.go

```diff
@@ -57,7 +57,7 @@ func GetMetadataResponse(testRuns []TestRun, log Logger, fetcher MetadataFetcher
 		productSpecs[i] = ProductSpec{ProductAtRevision: run.ProductAtRevision, Labels: run.LabelsSet()}
 	}
 
-	metadata, err := GetMetadataByteMap(log, fetcher)
+	_, metadata, err := GetMetadataByteMap(log, fetcher)
```

Comment:
Agree. It will be hard to add SHA with the current format, (`MetadataResults` is a map). I have filed https://github.com/web-platform-tests/wpt.fyi/issues/1938 and added TODOs